trip_app/trip_app/settings/base.py

```diff
@@ -31,131 +31,127 @@ def get_env_variable(var_name):
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'gdgh$)yv7v8*wuiw^0hk5l(9e57&=b1giij#5hq6=$5732y=95'
+SECRET_KEY = "gdgh$)yv7v8*wuiw^0hk5l(9e57&=b1giij#5hq6=$5732y=95"
```

Comment:
Hide this to .env/.conf/keys.yaml or to something like jks. Choose what you like.

Reply:
Thanks. Removed this to the env variables.